packages/cli/src/lib/index.js

```diff
@@ -120,6 +122,23 @@ export const getPaths = () => {
   }
 }
 
+/**
+ * Format a given string using the config provided in `prettier.config.js`
+ * of a Redwood project.
+ *
+ * @param {string} content - generated content.
+ */
+export const formatTemplate = (content) => {
+  try {
+    const { base: BASE_DIR } = getPaths()
+    const prettierConfig = require(path.join(BASE_DIR, 'prettier.config.js'))
```

Comment:
Moving this into a separate function:

```suggestion
    const prettierOptions = () => { 
      try {
        return require(path.join(getPaths().base, 'prettier.config.js'))
      } catch (e) {
        return undefined
      }
    }
```

Reply:
This makes more sense. I'll update it!